packages/relic/test/headers/special/accept_encoding_header_test.dart

```diff
@@ -0,0 +1,388 @@
+import 'dart:io';
```

Comment:
Question: Why are the headers folder named "special"?  Is that a common term? These headers seems to be more of the "standard" heeaders that are accepted by browsers.

What is the difference bewteen these and "basic" headers? Should they all be in a "standard" header folder alongside the "custom" folder?